src/core/kconfigelektra.cpp

```diff
@@ -0,0 +1,337 @@
+//
+// Created by felix on 28.10.19.
+//
+
+#ifdef FEAT_ELEKTRA
+
+#include "kconfigelektra_p.h"
+#include "kconfigdata.h"
+
+#include <QDir>
+#include <utility>
+#include <iostream>
+
+using namespace kdb;
+
+/**
+ * Necessary because kconfig only loads configuration if a file exists, if the file does not exist, nothing is loaded.
+ *
+ * It is possible that no user file exists, but a system file exists. If we were to simply use the user namespace,
+ * the system keys would be ignored.
+ *
+ * @param ks
+ * @return
+ */
+static std::string findLowestKeyspace(KeySet* ks)
```

Comment:
This should be automatically done when you use cascading lookups. (lookup key starts with `/`)